src/common/playlists/PlayList.jsx

```diff
@@ -24,7 +27,7 @@ const PlayList = () => {
     return (
       <div className='play-not-found'>
         <ImageOops className='play-not-found-image' />
-        <p className='page-404-lead'>Play not found</p>
+        <p className='page-404-lead'>Play not found for "{ searchTerm }" </p>
```

Comment:
@vasantisuthar You also need to handle for the filter criteria. For filter I would suggest the text to be "matched filter criteria" than the selected values.